Ports/ninja/package.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash ../.port_include.sh
+port=ninja
+version=1.8.2
+workdir=ninja-1.8.2
+files="https://github.com/ninja-build/ninja/archive/v1.8.2.tar.gz ninja-v1.8.2.tar.gz"
+
+build() {
+    CXX=i686-pc-serenity-g++ CXXFLAGS=--sysroot=${SERENITY_ROOT}/Build/Root \
+    LDFLAGS=--sysroot=${SERENITY_ROOT}/Build/Root \
```

Comment:
Always quote `SERENITY_ROOT` — I happen to have it in `/home/sergey/dev/serenity`, which doesn't contain any special characters, but others may not. (By the way, this wouldn't be a problem in our Shell :slightly_smiling_face:)

Reply:
```
$ rg SERENITY_ROOT .. | grep -v '"' | grep -v ninja | wc -l
14
```
:thinking:

Reply:
Clearly, some code does it wrong, doesn't mean we should add more code that does it wrong :slightly_smiling_face: